src/houston/views/index.pug

```diff
@@ -41,38 +41,42 @@ block body
 
   div.grid: div.two-thirds
     img(src='https://elementary.io/images/icons/places/128/distributor-logo.svg')
-    h1 Build for 
+    h1 Build for
```

Comment:
My editor is configured to trim trailing spaces, so that’s where they went. It’s fixed now.